Fix the compile_file context option

mociepka opened the pull request:
Fixes #21

Commit:
Add required image_path to context options

Issue #21

Commit:
Test the compile_file method

Issue #21

Commit:
Add test suit to the setup.py file

Now you can call `./setup.py test`.

Commit:
PEP8

bwhmather commented:
Should comment that https://github.com/mociepka/python-scss/commit/b9a6aca7201ad355c24d281f87238db8b590a8cc is worth pulling and making a new release for as it fixes the `std::logic_error` bug which entirely prevents `compile_file` from running.

Commit:
Use libsass version with fixed if statement

More details on: https://github.com/twbs/bootstrap-sass/issues/506
Now we are using the same libsass version as the node sass.

Commit:
Python 3 compatibility

Commit:
Remove test.pyc file

PhilipGarnero commented:
When is this going to be pulled ?
I need the image_path fix for compile_file.
Thanks anyway

PhilipGarnero commented:
@pistolero are you planning to merge this in the near future ?
